src/main/resources/de/theit/jenkins/crowd/CrowdSecurityRealm/CacheConfiguration/config.properties

```diff
@@ -25,3 +25,4 @@
 
 crowd.cache.size=Used cache size
 crowd.cache.TTL=Cache records timeout
+crowd.useTokenCache=Cache SSO authentication tokens
```

Comment:
This is the wrong place to do this
Here are located only a value located in DIV, which unfolds itself after checking use cache.